test/libsolidity/syntaxTests/pragma/experimental_solidity_out_of_order_2.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.8.19;
```

Comment:
Shouldn't this be removed?

Reply:
It should :-).

Reply:
Based on this comment: https://github.com/ethereum/solidity/pull/14168#discussion_r1193675019